iceoryx_posh/include/iceoryx_posh/popo/base_subscriber.hpp

```diff
@@ -142,7 +141,8 @@ class BaseSubscriber
 
   protected:
     BaseSubscriber() noexcept; // Required for testing.
-    BaseSubscriber(const capro::ServiceDescription& service) noexcept;
+    BaseSubscriber(const capro::ServiceDescription& service,
+                   const SubscriberOptions& subscriberOptions = SubscriberOptions()) noexcept;
```

Comment:
Why do we have a default for the subscriber but not the publisher? Do we need a default in `BaseSubscriber`?

Reply:
I did it on Thursday, which has an odd number of `u`, hence the default value. Since we now have Friday, it's removed